content/doc/tutorials/tutorial-for-installing-jenkins-on-IBM-Cloud.md

```diff
@@ -0,0 +1,179 @@
+# Get Jenkins on IBM Cloud
+
+You should have an IBM Cloud account, otherwise you can [register here].
+At the end of the tutorial you will have a cluster with Jenkins up and runnning.
+
+1. We will provision a new Kubernetes Cluster for you if, you already have one skip to step **2**
+2. We will deploy  the IBM Cloud Block Storage plug-in, if already have it skip to step **3**
+3. Jenkins deployment
+
+## Step 1 provision Kubernetes Cluster
+
+* Click the **Catalog** button on the top 
+* Select **Service** from the catalog
+* Search for **Kubernetes Service** and click on it
+![Kubernetes](/content/images/IBMCloud/kubernetes-select.png)
+* You are now at the Kubernetes deployment page, you need to specify some details about the cluster 
+* Choose a plan **standard** or **free**, the free plan only has one worker node and no subnet, to provision a standard cluster, you will need to upgrade you account to Pay-As-You-Go 
+  * To upgrade to a Pay-As-You-Go account, complete the following steps:
+
+  * In the console, go to Manage > Account.
+  * Select Account settings, and click Add credit card.
+  * Enter your payment information, click Next, and submit your information
+* Choose **classic** or **VPC**, read the [docs] and choose the most suitable type for yourself 
+ ![VPC](/content/images/IBMCloud/infra-select.png)
+* Now choose your location settings, for more information please visit [Locations]
+  * Choose **Geography** (continent)
+![continent](/content/images/IBMCloud/location-geo.png)
+  * Choose **Single** or **Multizone**, in single zone your data is only kept in on datacenter, on the other hand with Multizone it is distributed to multiple zones, thus  safer in an unforseen zone failure 
+![avail](/content/images/IBMCloud/location-avail.png)
+  * Choose a **Worker Zone** if using Single zones or **Metro** if Multizone
+ ![worker](/content/images/IBMCloud/location-worker.png) 
+    * If you wish to use Multizone please set up your account with [VRF] or [enable Vlan spanning]
+    * If at your current location selection, there is no available Virtual LAN, a new Vlan will be created for you 
+ 
+* Choose a **Worker node setup** or use the preselected one, set **Worker node amount per zone**
+![worker-pool](/content/images/IBMCloud/worker-pool.png)
+* Choose **Master Service Endpoint**,  In VRF-enabled accounts, you can choose private-only to make your master accessible on the private network or via VPN tunnel. Choose public-only to make your master publicly accessible. When you have a VRF-enabled account, your cluster is set up by default to use both private and public endpoints. For more information visit [endpoints].
```

Comment:
Jenkins has deprecated the term `master` in favor of `controller`.  Conversion of terminology is a work in progress
```suggestion
* Choose **Master Service Endpoint**,  In VRF-enabled accounts, you can choose private-only to make your controller accessible on the private network or via VPN tunnel. Choose public-only to make your controller publicly accessible. When you have a VRF-enabled account, your cluster is set up by default to use both private and public endpoints. For more information visit [endpoints].
```